browsers/nodejs.json

```diff
@@ -3,6 +3,24 @@
     "nodejs": {
       "name": "Node.js",
       "releases": {
+         "0.1.100": {
```

Comment:
```suggestion
        "0.1.100": {
```

Reply:
This PR has already been merged.  We can come back to this in #3737, where we'll be enforcing styling anyways.  :wink: